docs/common-domain-model.md

```diff
@@ -18,6 +18,6 @@ In each section, selected examples of model definitions are used as
 illustrations to help explain each dimension and include, where
 applicable, data samples to help demonstrate the structure.
 
-The CDM is expressed in a language called the Rosetta DSL. All the
+The CDM is expressed in a language called the Rune DSL. All the
 language components used by the CDM including types, functions and
-annotations are described in the [Rosetta DSL Documentation](https://docs.rosetta-technology.io/rosetta/rosetta-dsl/rosetta-modelling-component).
+annotations are described in the [Rune DSL Documentation](https://docs.rosetta-technology.io/rosetta/rosetta-dsl/rosetta-modelling-component).
```

Comment:
Link is wrong too:
https://docs.rosetta-technology.io/rosetta/rosetta-dsl/rune-modelling-component/